### ISSUE TITLE
About image preprocessing

### ISSUE THREAD

**Issue by Hanawh** [isht7/pytorch-deeplab-resnet#49]
If I want to implement deeplabv2 in my own task，should the image preprocessing be same？like no padding，just resize

**Comment by isht7** [comment]
What are your image sizes in the train set and in the test set?

**Comment by Hanawh** [comment]
> What are your image sizes in the train set and in the test set?

[640，480] resize and pad to [320，320]
